clients/CMakeLists.txt

```diff
@@ -77,13 +77,16 @@ if( BUILD_CLIENTS_BENCHMARKS OR BUILD_CLIENTS_TESTS)
   if( NOT WIN32 )
     # Linking lapack library requires fortran flags
     find_package( cblas REQUIRED CONFIG )
-    if (LINK_BLIS)
-      set( BLIS_INCLUDE_DIR ${BUILD_DIR}/deps/blis/include/blis )
```

Comment:
Not sure what you are doing here you are in a NOT WIN32 already.   Are you dropping blis?

Reply:
It doesn't appear that we're using blis at all. I changed this code back to what it was, but we aren't defining LINK_BLIS in install.sh or rmake.py so it isn't used anyway. I'll work on adding blis in another PR.

Reply:
So what is the goal of this PR, rmake.py should work find on linux?  Yes blis or other reference another PR.  OpenBLAS on windows covers rocblas (solver have to check?), on linux should get solver input as well.

Reply:
Yes, rmake.py now works on linux after this PR.